src/components/panel/panel.js

```diff
@@ -1175,10 +1130,9 @@ MdPanelRef.prototype._focusOnOpen = function() {
     // Wait a digest to guarantee md-autofocus has finished adding the class
```

Comment:
Update this comment? We are waiting for angular to finish rendering the template now rather than a digest.